starter-code/css/main.css

```diff
@@ -1,8 +1,112 @@
 /*
 Colors:
-
 Text: 1A1A1A
 Green: #00B172
 White: #FFF
-
 */
+
+*{
+	border: 1px solid transparent;	
+	box-sizing: border-box;
+	font-family: 'Lato', sans-serif;
+}
+
+ul{
+	float: right;
+	padding-right: 50px;
+}
+
+li{
+	list-style: none;
+	display: inline;
+    margin-left: 10px;
+}
+
+nav{
+	height: 100px;
+	margin-bottom: 40px;
+/*	position: static;
+*/}
+
+.container{
+	text-align: center;
+}
+
+
+.picture{
+	background-image:url(../images/landing.jpg);
+	background-size: cover;
+	background-attachment: unset;
+    background-position: center;
+
+/*	background-color: blue;
+*/	height: 700px;
+}
+
+.picture h1, .picture p{
+	color: white;
+    width: 400px;
```

Comment:
There is no need to set a fixed width to those elements. You can remove `width: 400px;`, `margin: 10px auto;` and `text-align: center;`. Then, to center, add `text-align: center;` to the *parent*, which is `.picture`.